modules/log.cpp

```diff
@@ -551,6 +555,74 @@ CModule::EModRet CLogMod::OnChanMsg(CNick& Nick, CChan& Channel,
     return CONTINUE;
 }
 
+bool CLogMod::OnWebRequest(CWebSock& WebSock, const CString& sPageName, CTemplate& Tmpl) {
+    if(GetType() == CModInfo::EModuleType::GlobalModule) {
+        return true;
+    }
+
+    CString sPath(WebSock.GetParam("path", false));
```

Comment:
Why not `=`?

Reply:
I saw other code using this syntax for triggering the CString constructor. I can switch to triggering it via assignment if that is preferred.

Reply:
It's the copy constructor from another CString. So I think it's more obvious with `CString sPath = ...;`

Reply:
Ok. I've revised the instances of where this occurred in my new code.